plugins/woocommerce-admin/client/core-profiler/components/navigation/navigation.tsx

```diff
@@ -0,0 +1,76 @@
+/**
+ * External dependencies
+ */
+import { __ } from '@wordpress/i18n';
+import classnames from 'classnames';
+import { Button } from '@wordpress/components';
+/**
+ * Internal dependencies
+ */
+import WooLogo from './woologo';
+import './navigation.scss';
+import ProgressBar from '../progress-bar/progress-bar';
+
+type NavigationProps = {
+	onSkip?: () => void;
+	percentage?: number;
+	previous?: string;
+	showProgress?: boolean;
+	showLogo?: boolean;
+	classNames?: { mobile?: boolean };
+	skipText?: string;
+	progressBarColor?: string;
+};
+
+const Navigation = ( {
```

Comment:
Just one last thing, what do you think if we use `export const` instead of `export default`?  It allows us to autocomplete the name `import { Navigation } from ...` when importing the component.

Reply:
Sounds good! Updated in https://github.com/woocommerce/woocommerce/pull/37741/commits/fe35543e9b1a0b763d773fbae7f091f582a41d25